<?xml version="1.0"?>
<launch>
    <arg name="ns1" default="wheelbird1"/>
    <arg name="ns2" default="wheelbird2"/>
    <arg name="ns3" default="wheelbird3"/>
    <arg name="ns4" default="wheelbird4"/>
    <arg name="ns5" default="wheelbird5"/>
    <arg name="ns6" default="wheelbird6"/>
    <rosparam command="load" file="$(find wheelbird_control)/config/mpc_ugv.yaml" />
    <param name="robot_name" type="string" value="$(arg ns5)"/>
    
    <group ns="$(arg ns5)">
        <node name="wheelbird5" pkg="wheelbird_control" type="mpc5_ground" output="screen">

	</node>
    </group>

</launch>